CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The version numbers below try to follow the conventions at http://semver.org/.
 
 ## Unreleased
 
+- Adding validades with presence uniqueness of ids
```

Comment:
What do you think about: `Adds validations (presence and uniqueness) of foreign keys`?